avr/cores/megacommand/MCL/MCL.cpp

```diff
@@ -43,8 +43,9 @@ void MCL::setup() {
 
   mcl_actions.setup();
   mcl_seq.setup();
-  MDSysexListener.setup(&Midi);
 
+  MDSysexListener.setup(&Midi);
+  trig_interface.setup(&Midi);
```

Comment:
Suggest:

Use 0x30 to query if TI mode is present.
Then multiplex md_exploit within trig interface?

Reply:
Yeah that might be a good idea.

Would be even better if users upgrade their machines  :)

Reply:
I've decided to deprecate md_exploit.

Having both systems will add complexity to the code in which I do not want to manage. 

For example, there are some scenarios whereby the exploit must be turned off, during (specific sysex commands) and then turned back on again.

Reply:
Also, it means we have to test both systems during fw upgrades. No thanks.

Reply:
I agree. Let's move forward.

Reply:
It also means that exploit-related bugs are all gone -- when you tweak the params, the TI no longer fails; no need to switch to track 16 anymore;